src/traverse/node-path.js

```diff
@@ -113,6 +113,33 @@ class NodePath {
   }
 
   /**
+   * Returns parent.
+   */
+  getParent() {
```

Comment:
That's said, this seems obsolete, since doesn't abstract anything. If you'd like to have `getParent` for consistency with `getChild` (with might be a good reason), let's change it to:

```js
/**
 * Returns parent path.
 */
getParent() {
  return this.parentPath;
}
```